classes/Cart.php

```diff
@@ -1655,6 +1655,24 @@ public function updateQty(
         return true;
     }
 
+    /**
+     * Returns the language related to the cart or the default one if deleted
```

Comment:
Should it be a method in the ObjectModel instead, by checking if id_lang exists? :thinking:

Reply:
(In order to avoid duplicate)

Reply:
I'm not sure, it seems weird to me to have a method `getLanguage()` for an object that doesn't have a Language.

Reply:
This is something we already do for:
`getHtmlFields`
 `uploadMultiShopTable`
`getAssociatedShops`
`getWebserviceParameters`
...

There are a lot of generic methods like this :sweat_smile: